ATP93-Košice.md

```diff
@@ -0,0 +1,16 @@
+# Košice, Slovakia
+
+### Restaurants
+
+- Republika východu
+- Camelot
+
+### Attractions / Places to see
+
+- St. Elisabeth cathedral
+- Peter's Park
+
+### Fun Nearby
+
+- Steel Arena
+- State Theatre
```

Comment:
I haven't been there for a while, but my favourite play used to be One Flew Over the Cuckoo's Nest